src/signals/incident/services/convert-value/index.ts

```diff
@@ -0,0 +1,27 @@
+import { isArray } from 'utils';
+
+const convertValue = (value: unknown = undefined, postPropertyNames: string[] = []): any => {
```

Comment:
The returnType of this function is also `unknown`

Reply:
Not sure if defaulting to undefined is useful, since by definition a prop that is not passed is undefined